updatePomVersions.sh

```diff
@@ -33,6 +29,15 @@ sbt publishM2;publishLocal
   git checkout */pom.xml
 )
 
+if [ "$1" = "all" ]; then
+  PROJ="samples/*java-*"
+elif [ $1 ]; then
+  PROJ=$1  
+else 
+  PROJ=""
```

Comment:
what happens in this case? should we throw an error and show how to invoke the script?

Reply:
When we choose to not pass an argument, we only publish the artifacts locally. 

But I think we should merge this script with the `publishLocalM2` one. 

So, no argument, only publish local. With arguments, updated the sample to run locally.

I never use the `publishLocalM2` script because I want to update the pom.xml and open the project in IntelliJ. 

Let me update this PR by merging both scripts.

Reply:
Would make sense just delete `publishLocalM2` and use only `updatePomVersion`?

Reply:
yep, I think we should have only one script. It's just that I keep postponing the task. I will do it one day. 😄